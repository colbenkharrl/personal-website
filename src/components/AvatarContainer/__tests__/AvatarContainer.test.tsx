import { render, screen } from '@testing-library/react';
import { AvatarContainer } from '../';

it('AvatarContainer matches snapshot (regression test)', () => {
  render(<AvatarContainer data-testid="AvatarContainer" />);
  expect(screen.getByTestId('AvatarContainer')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
