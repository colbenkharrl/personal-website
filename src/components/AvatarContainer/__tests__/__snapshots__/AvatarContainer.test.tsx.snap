// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvatarContainer matches snapshot (regression test): Full component snapshot. 1`] = `
<div
  class="h-10 w-10 rounded-full bg-white/90 p-0.5 shadow-lg shadow-zinc-800/5 ring-1 ring-zinc-900/5 backdrop-blur dark:bg-zinc-800/90 dark:ring-white/10"
  data-testid="AvatarContainer"
/>
`;
